conformance/build.gradle.kts

```diff
@@ -4,13 +4,38 @@ import net.ltgt.gradle.errorprone.errorprone
 plugins {
     `version-catalog`
 
+    application
     java
     alias(libs.plugins.errorprone)
 }
 
+val conformanceCLIFile = project.layout.buildDirectory.file("gobin/protovalidate-conformance").get().asFile
+val conformanceCLIPath: String = conformanceCLIFile.absolutePath
+val conformanceAppScript: String = project.layout.buildDirectory.file("install/conformance/bin/conformance").get().asFile.absolutePath
+val conformanceArgs = (project.findProperty("protovalidate.conformance.args")?.toString() ?: "").split("\\s+".toRegex())
+
+tasks.register<Exec>("installProtovalidateConformance") {
+    description = "Installs the Protovalidate Conformance CLI."
+    environment("GOBIN", conformanceCLIFile.parentFile.absolutePath)
+    outputs.file(conformanceCLIFile)
+    commandLine(
+        "go",
+        "install",
+        "github.com/bufbuild/protovalidate/tools/protovalidate-conformance@${project.findProperty("protovalidate.version")}",
+    )
+}
+
+tasks.register<Exec>("conformance") {
```

Comment:
This logic was specific to this subdirectory (and project) but was included in the top-level makefile before. It helps to move it closer to where it is used.